pkgs/tools/X11/keynav/default.nix

```diff
@@ -1,9 +1,10 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, libX11, xorgproto, libXtst, libXi, libXext
 , libXinerama, libXrandr, glib, cairo, xdotool }:
 
-let release = "20180821"; in
-stdenv.mkDerivation {
-  name = "keynav-0.${release}.0";
+stdenv.mkDerivation rec{
```

Comment:
Missing a whitespace after `rec`. Also why pull `release` into the attribute set if it could just stay in the `let`-binding? We don't use it as environment variable within the build (which we could) so this is really not improving anything.

Reply:
reverted the changes

Reply:
Still missing the whitespace and the rec is still there.